src/TipBox.vue

```diff
@@ -1,10 +1,12 @@
 <template>
     <div class="alert container" :class="[boxStyle, addClass]" :style="customStyle">
         <div v-if="!isDefault" class="icon-wrapper">
-            <span v-html="iconType"></span>
+            <slot name="i">
```

Comment:
Don't think it is good to use single letters as names for the slots. :P

If it is meant to be used internally only, maybe it can start with an underscroll? Like `_icon`.